dependencies.props

```diff
@@ -46,7 +46,7 @@
   <!-- Package dependency verification/auto-upgrade configuration. -->
   <PropertyGroup>
     <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
-    <DependencyBranch>master</DependencyBranch>
+    <DependencyBranch>release/uwp6.0</DependencyBranch>
```

Comment:
Will all of the `RemoteDependencyBuildInfo`s have a `release/uwp6.0` branch? Might need to split this property into two if there are some products that won't have an explicit branch for uwp6.0.